presto-resource-group-managers/src/main/java/com/facebook/presto/resourceGroups/db/DbResourceGroupConfigurationManager.java

```diff
@@ -119,6 +120,15 @@ public DbResourceGroupConfigurationManager(ClusterMemoryPoolManager memoryPoolMa
         if (lastRefresh.get() == 0) {
             throw new PrestoException(CONFIGURATION_UNAVAILABLE, "Root groups cannot be fetched from database");
         }
+
+        if (System.nanoTime() - lastRefresh.get() > maxRefreshInterval.toMillis() * 1_000_000) {
+            // TODO: consider not throwing exception but only have a warning in this case
```

Comment:
Why this is a todo? It doesn't sound like something hard to do technically. If it's because we are not sure whether exception makes more sense or warning makes more sense, either create an issue to evaluate this, or add that as a config option so both are possible. Adding this kind of TODO in code will just leave them here forever without resolution, in that case you might as well just remove this TODO.

Reply:
@rongrong : I can open an issue in that case I guess :). Just want to have some kind of tracking :) .

Technically staled resource group config is not severe enough to fail the query. However we do want to make sure there is some way to throw a warning (instead of go through everything silently).

Anyhow, I will remove the TODO for now and open an issue for the tracking purpose ;)